### PR TITLE
Add glossary for shared language

### PR DESCRIPTION
**Pull request by adborden**
Mostly for our own team.

### CODE REVIEW

File: Glossary.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+The Freedom of Information Act which provides that any person has a right,
+enforceable in court, to obtain access to federal agency records, except to the
+extent that any portions of such records are protected from public disclosure by
+one of nine exemptions or by one of three special law enforcement record
+exclusions.
+
+
+##### FOIA Public Liaison
+
+Designated by the Chief FOIA Officer who is responsible for acting as the public
```
Comment (pkarman) [inline Glossary.md]: The "..who is responsible" is ambiguous. Maybe instead

> Responsible for acting as the public contact for addressing concerns of requesters related to FOIA administration. Designated by the Chief FOIA Officer.

File: Glossary.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+##### release to three, release to all
+
+
+##### requester
+
+A person or organization making or about to make a FOIA request.
+
+
+##### response
```
Comment (pkarman) [inline Glossary.md:85]: Maybe add "responsive document" entry here too?

File: Glossary.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+##### portal
+
+A gateway to other systems or information but may not have any functionality or
+information on its own.
+
+
+##### proactive disclosure
+
+
+##### release to three, release to all
```
Comment (pkarman) [inline Glossary.md:72]: Per the FOIA 1996 amendment, if a set of records is requested three times, the FOIA indicates the documents should be made permanently public, which forms the basis of the [“reading room” provision](https://www.justice.gov/oip/blog/foia-post-2003-foia-counselor-qa-frequently-requested-records).

**Comment by pkarman** [comment]
@adborden thanks for doing this -- great idea to help reduce blocking effect of jargon.

Made some suggestions for edits and entries.

File: Glossary.md [hunk 4]
```diff
@@ -71,6 +71,10 @@ information on its own.
 
 ##### release to three, release to all
 
+Per the FOIA 1996 amendment, if a set of records is requested three times, the
+FOIA indicates the documents should be made permanently public, which forms the
+basis of the “reading room” provision.
```
Comment (pkarman) [inline Glossary.md]: thanks. maybe add the link
```markdown
[“reading room” provision](https://www.justice.gov/oip/blog/foia-post-2003-foia-counselor-qa-frequently-requested-records)
```

and then good to go.

**Review by pkarman — APPROVED** [review]
lgtm % comments

File: Glossary.md (hunk 4 above)
Comment (adborden) [reply Glossary.md]: Oops, yep, sorry I missed that.

**Pull request merged**